docs/OMS-Agent-for-Linux.md

```diff
@@ -137,14 +137,17 @@ The proxy configuration value has the following syntax:
 Property|Description
 -|-
 Protocol|http or https
-user|Optional username for proxy authentication
-password|Optional password for proxy authentication
+user|username for proxy authentication
+password|password for proxy authentication
 proxyhost|Address or FQDN of the proxy server
 port|Optional port number for the proxy server
 
 For example:
 http://user01:password@proxy01.contoso.com:8080
 
+*Note: Although you do not have any user/password set for the proxy, you will still need to add a psuedo user/password. This can be any username or password.    
+(This will be enhanced in future so that these psuedo user/password will not be necessary)*
```

Comment:
nit: phrase it like so: "This will be enhanced in the future so that a psuedo username/password will not be necessary."